src/Backoffice/Airline/App/Request/UpdateAirlineRequest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Airline\App\Request;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+use Lightit\Backoffice\Airline\Domain\DataTransferModels\AirlineDto;
+use Lightit\Backoffice\Airline\Domain\Models\Airline;
+
+class UpdateAirlineRequest extends FormRequest
+{
+    public const NAME = 'name';
+
+    public const DESCRIPTION = 'description';
+
+    public const CITIES = 'cities';
+
+    public function rules(): array
+    {
+        /** @var Airline $airline */
+        $airline = $this->airline;
+
+        return [
+            self::NAME => ['required', 'string', 'max:255', Rule::unique('airlines')->ignore($airline->id)],
+            self::DESCRIPTION => ['required', 'string'],
+            self::CITIES => ['required', 'array'],
+            self::CITIES . '.*' => ['required', 'array'],
```

Comment:
I think this is unnecessary 
```suggestion
            self::CITIES => ['required', 'array'],
```